init.ts

```diff
@@ -138,6 +138,7 @@ async function onInitHooks() {
 	});
 
 	// this will pre-fetch the device types and populate the cache...
+	// eslint-disable-next-line @typescript-eslint/no-floating-promises
 	getAccessibleDeviceTypes(sbvrUtils.api.resin);
```

Comment:
Wouldn't awaiting here create a guarantee that we may need?

Reply:
We don't need to await this, since slow or non-accessible S3 shouldn't block the API startup.
As per the pre-existing comment `// this will pre-fetch the device types and populate the cache...` this is here only to warm-up the device-type.json cache.

Reply:
Let me update the comment